### DISCUSSION TITLE
Schema name changes from dbo.dbName to QN@dbName and auto-complete stops working after first query Run

### DISCUSSION THREAD

**Discussion by SashaSherman-QA** [dbeaver/dbeaver#20222] [Q&A]
Hi,
Why schema name changes to QN@dbName instead dbo.dbName after I run the query (which is working and auto-complete is ok)
But when I start to type second query (after I ran first) auto complete does not work and QN@dbName appears as schema name (in toolbar)
Also my update permissions are blocked, but they work fine in MSSQL Studio
Only if I write the whole path - auto-complete starts working:  select * from dbo.dbName.table...
(the "dbName"  was set to default previously - if it's relevant)
Happened after I upgrated to last version today

**Comment by LonwoLonwo** [comment]
Hello @SashaSherman-QA 

Please provide footage of your issue.

And does disabling this setting help in your case?

![2023-06-07 20_07_13-Preferences](https://github.com/dbeaver/dbeaver/assets/45152336/cac2885c-fb08-4c06-b5bb-c7bd16aaa328)

**Comment by SashaSherman-QA** [comment]
Disabling the experimental features didnt work, I also uninstalled and installed again selecting Reset settings option..no success.
Problem remains:
1. **Immediately after opening DBeaver** auto complete works and the schema in toolbar on top is correct: **dbo@GlobalE**
    I open new SQL script and enter the query select * from orders - auto complete works ok
    ![image](https://github.com/dbeaver/dbeaver/assets/95393340/cc91b895-b378-4e23-b8a2-969f72037109)
    
2.  **After running the query once:** schema name changes automatically to **QN@GlobalE** and auto-complete stops working, 
     when I try to type another query
     ![image](https://github.com/dbeaver/dbeaver/assets/95393340/64cf2e39-8746-4710-b979-813cf2ed5b22)
3. in Database GlobalE - "dbo" is set as default Schema - why does it switch to some other QN just after one query run?
    And from toolbar - I cant see other schemas and switch manually to dbo@GlobalE

    ![image](https://github.com/dbeaver/dbeaver/assets/95393340/0162c948-ce49-423d-a93e-e2fe98527bcd)

**Reply by SashaSherman-QA** [reply]
Hi,
I tried with other user - it worked ok for me (problem didnt reproduce)
hope it will work ok in future...
By the way the reason I updated to new version was - that suddenly few days ago when I tried to open DBeaver - I got Error 
"JRE or JDK is not found" - out of noowhere
Just for your information (Reinstall solved the issue)

Thanks, sorry for taking your time.
Have a good day!
Sasha
